250_bgvideo.diff added a demux_open,
and dvdnav added another parameter.

diff -Nur MPlayer-20060519.orig/mplayer.c MPlayer-20060519/mplayer.c
--- MPlayer-20060519.orig/mplayer.c	2006-06-02 19:26:22.000000000 +0300
+++ MPlayer-20060519/mplayer.c	2006-06-02 19:28:19.000000000 +0300
@@ -3305,7 +3305,7 @@
     mp_msg(MSGT_CPLAYER,MSGL_ERR, "Couldn't open the stream for back ground video: %s\n",bg_video);
     break;
   }
-  bg_demuxer = demux_open(bg_s,bg_file_format,-2,video_id,dvdsub_id,bg_video);
+  bg_demuxer = demux_open(bg_s,bg_file_format,-2,video_id,dvdsub_id,bg_video, 0);
   if(!bg_demuxer) {
     mp_msg(MSGT_DEMUXER,MSGL_ERR,"Failed to open background video\n");
     free_stream(bg_s);
